From 2b7c1320bcaeeb6f3cf6843aa313254cd8b61d1d Mon Sep 17 00:00:00 2001
From: deepongi <infectedmushi@gmail.com>
Date: Wed, 3 Dec 2025 01:34:05 +0000
Subject: [PATCH] 20-KernelSU-Spoof-LKM-mode-to-suppress-manager-warning

---
 kernel/supercalls.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/supercalls.c b/kernel/supercalls.c
index 64405ea4..fa4c3746 100644
--- a/kernel/supercalls.c
+++ b/kernel/supercalls.c
@@ -78,7 +78,7 @@ static int do_get_info(void __user *arg)
 {
     struct ksu_get_info_cmd cmd = {.version = KERNEL_SU_VERSION, .flags = 0};
 
-#ifdef MODULE
+#if defined(MODULE) || defined(CONFIG_KSU_SUSFS)
     cmd.flags |= 0x1;
 #endif
 
@@ -950,4 +950,4 @@ int ksu_install_fd(void)
     pr_info("ksu fd installed: %d for pid %d\n", fd, current->pid);
 
     return fd;
-}
\ No newline at end of file
+}
-- 
2.52.0

